0004/README.md

```diff
@@ -0,0 +1,416 @@
+The OpenSSF will run a community-maintained repository of approved auditors and
+audited build platforms. This repository may be the same one used for the
+self-attestation tier.  
+In order to participate in this tier, build platforms should
+
+1. Successfully complete an audit with an approved auditor
+1. Create a PR for the repository containing
+    1. Build platform name
+    1. Audited SLSA levels
+    1. URI that resolves to the public audit report
+    1. (Optional) Build ID
+    1. (Optional) List of URIs that resolve to proof of SLSA conformance
+    1. (Optional) List of URIs that resolve to materials for verifying
+        provenance
+    1. (Optional) List of URIs that resolve to the build platform's
+        public vulnerability disclosures
+
+1. Complete the terms and conditions form.
+1. Send the PR for review.
+1. Have the auditor approve the PR with a comment indicating the audited
+    SLSA levels
+
+The maintainers will review the PR and merge it after the auditor's approval.
+
+Once the PR is merged and the terms are signed, the OpenSSF grants the build
+platform a limited license to use the SLSA logo on its website and other
+promotional materials. Should the build platform fail to adhere to the program's
+terms or meet its requirements, the OpenSSF will revoke that license and, if
+necessary, enforce its trademarks. 
+
+### Auditor participation
+
+Auditors can take part in the program by:
+
+1. Registering their intent with the OpenSSF.
+1. Performing any training and/or certification required by the OpenSSF (TBD)
+1. Completing the terms and conditions form.
+1. Opening a PR against the auditor repository containing 
+    1. Name of the auditor
+    1. Auditor's website
+    1. A URI that resolves to the auditor's auditing procedure. 
+
+The community will review the PR and merge it once it has been approved by a
+quorum of maintainers. It is not yet clear what, if any, criteria will be used
+to evaluate auditors.   
+
+Once the PR is merged and the terms and signed, the OpenSSF grants the auditor a
+license to use the SLSA logo on its website and promotional materials. The
+OpenSSF reserves the right to revoke the license if the auditor fails to
+publicize audit reports,  performs insufficiently thorough audits, or otherwise
+acts against the SLSA Community's interests.  
+
+## Alternatives considered
+
+### Central repository
+
+The OpenSSF would maintain a repository that stored all public proof of
+conformance and provenance verification materials for build platforms taking
+part in the program. This option made enforcement easy since you could remove
+build platforms that do not comply with the program's terms, but it puts too
+much trust in the repository's maintainers. This option would also require
+infrastructure for build platforms to rotate and revoke provenance verification
+materials, which would complicate the design.
+
+### Build platform attestations
```

Comment:
You could still provide it as a separate attestation document entirely and one that doesn't necessarily need to be updated all the time. Even if we go the human readable route, I do wonder if folks are going to want some machine readable content to help with some of the work or for internal consumption for compliance.

e.g. the same human readable content just as key/values in a JSON that could then be ingested into an organizations risk management/governance/compliance platform.

Reply:
How about we require that proof of conformance docs and audit reports be available in a computer-readable format? That way they're easy to manipulate, but we aren't passing them around in the provenance.